ipv6/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator:
+- behavior: replace
+  literals:
+  - PROVISIONING_IP=fd2e:6f44:5dd8:b856::2
+  - DEPLOY_KERNEL_URL=http://[fd2e:6f44:5dd8:b856::2]:6180/images/ironic-python-agent.kernel
+  - DEPLOY_RAMDISK_URL=http://[fd2e:6f44:5dd8:b856::2]:6180/images/ironic-python-agent.initramfs
+  - IRONIC_ENDPOINT=http://[fd2e:6f44:5dd8:b856::2]:6385/v1/
+  - IRONIC_INSPECTOR_ENDPOINT=http://[fd2e:6f44:5dd8:b856::2]:5050/v1/
+  - CACHEURL=http://[fd2e:6f44:5dd8:b856::1]/images
+  name: ironic-bmo-configmap
+resources:
+- ../../go/src/github.com/metal3-io/baremetal-operator/deploy
```

Comment:
This assumes dev-scripts is checked out in $HOME, unfortunately kustomize doesn't have a way to change the base path easily, might need to be a sed thing or generate this whole file in a script.

Reply:
resources can't take an absolute path?

Reply:
Oh I see even with an absolute path we'll need a variable to say where the `$GOPATH` is